### PR TITLE
JavaScript library has changed name from nexmo-stitch to nexmo-client.

### PR DESCRIPTION
**Pull request by tbedford**
## Description

The JavaScript library was renamed. I've renamed from `nexmo-stitch` to `nexmo-client` in the main docs.

### CODE REVIEW

**Comment by AlexLakatos** [comment]
Let's hold off on merging this. The new library was released only today, and it contains breaking changes, need to check if the building blocks are still correct and if the quickstarts still work. I'll also need to approve and merge the new generated js sdk docs.
I'd like to update the code as well as the quickstarts in one go, so I'll take this on next week

**Comment by mheap** [comment]
@AlexLakatos Do we have any ETA on this work being done?

**Comment by AlexLakatos** [comment]
hopefully today or tomorrow

**Comment by mheap** [comment]
@AlexLakatos Following up on this. Any progress?

**Comment by mheap** [comment]
@AlexLakatos Ping

**Comment by AlexLakatos** [comment]
removed the broken guides, and updated client fro stitch everywhere, added the new docs. I think it's good to go now, please review.

File: .repos/nexmo/conversation-docs/ios/0.2.64/Classes/NXMConversationEventsController.html [hunk 1]
```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
```
Comment (tbedford) [inline .repos/nexmo/conversation-docs/ios/0.2.64/Classes/NXMConversationEventsController.html]: @AlexLakatos Was the addition of these files to the PR by design?

Comment (AlexLakatos) [reply .repos/nexmo/conversation-docs/ios/0.2.64/Classes/NXMConversationEventsController.html]: Yeah, they are the generated SDK docs from the client sdk code.

**Comment by tbedford** [comment]
LGTM. Please feel free to merge ( I can't approve and merge as I created the PR). Thanks!

**Comment by AlexLakatos** [comment]
Thanks. I'll deploy a test app and merge after I check it.

**Pull request merged**